flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/TypeInformationAnyType.java

```diff
@@ -68,6 +68,10 @@ public TypeInformationAnyType(TypeInformation<T> typeInfo) {
 		this(true, typeInfo);
 	}
 
+	public TypeInformationAnyType(Class<T> clazz) {
```

Comment:
This does not belong into a logical type but into some utility functions. It calls the type extractor and can potentially throw exceptions that need to be catched.

Reply:
Good catch, I'll take it out.